leaf-ui/rappid-extensions/AssignmentsTable.js

```diff
@@ -172,10 +175,12 @@ var AssignmentsTable = Backbone.View.extend({
     displayAbsoluteRelationshipAssignments: function(){
         this.model.getLinks().forEach(linkCell => {
             linkBbm = linkCell.get('link');
-            if(linkBbm.isValidAbsoluteRelationship()){
-                var linkRelationshipView = new LinkRelationshipView({model: linkBbm});
-                $('#link-list').append(linkRelationshipView.el);
-                linkRelationshipView.render();
+            if(linkCell.isValidAbsoluteRelationship()){
+                var sourceName = linkCell.getSourceElement().get('intention').get('nodeName');
```

Comment:
Since you only use these variables once you can inline them (aka linkSrc: linkCell.getSourceElement().get('intention').get('nodeName'))